x-pack/plugins/timelines/common/types/timeline/actions/index.ts

```diff
@@ -100,4 +100,7 @@ export interface BulkActionsObjectProp {
 }
 export type BulkActionsProp = boolean | BulkActionsObjectProp;
 
-export type AlertStatus = 'open' | 'closed' | 'in-progress';
+export type AlertWorkflowStatus = 'open' | 'closed' | 'acknowledged';
```

Comment:
Doesn't this duplicate the runtime type you defined in the `observability` plugin. Can that runtime type be put here and just imported into the `observability` plugin?

Reply:
let me see if the runtime types are used at all

Reply:
Should all of this be moved to the `rule_registry` or `alerting` plugins, or to `@kbn/rule-data-utils` ?

@dplumlee You added/modified the types in the `timelines` plugin. What do you think?

Reply:
Sure, a separate PR is fine.

In #108679 I'm putting the constants for the `ALERT_STATUS` into ` packages/kbn-rule-data-utils/src/alerts_as_data_rbac.ts`. Looks like that would make sense for these types too?

Reply:
yeah that would make sense 👍